service/rpc/state.go

```diff
@@ -75,6 +78,11 @@ type cancelUnbondRequest struct {
 	GasLimit uint64 `json:"gas_limit"`
 }
 
+type queryRedelegationRequest struct {
```

Comment:
`queryRedelegationsRequest`
```suggestion
type queryRedelegationsRequest struct {
```

Reply:
Also renamed `queryRedelegationEndpoint` to `queryRedelegationsEndpoint`